nixos/doc/manual/development/unit-handling.section.md

```diff
@@ -48,10 +48,11 @@ checks:
     If the flag is enabled, the unit is **stop**ped and then **start**ed. If
```

Comment:
while you're at it, could you please also change 

```diff
-  - The rest of the behavior is decided whether the unit has `X-StopIfChanged`
-     in the `[Service]` section set (exposed via
+  - Further behavior depends on the unit having `X-StopIfChanged` set in the
+    `[Service]` section (exposed via
```

on line 44-45 (cannot mark it here as it doesn't belong to the change)